pom.xml

```diff
@@ -104,6 +104,15 @@
                         </exclude>
                     </configuration>
                 </plugin> 
+                <plugin>
```

Comment:
I don't think this change is needed when using the 1.0.5 parent pom.

Reply:
correct.

Reply:
Done.